src/main/java/com/zendesk/maxwell/Maxwell.java

```diff
@@ -55,15 +41,8 @@ private void run(String[] argv) throws Exception {
 
 			SchemaStoreSchema.handleMasterChange(schemaConnection, context.getServerID(), this.config.databaseName);
 
-			if ( this.context.getInitialPosition() != null ) {
-				String producerClass = this.context.getProducer().getClass().getSimpleName();
-
-				LOGGER.info("Maxwell is booting (" + producerClass + "), starting at " + this.context.getInitialPosition());
-
-				this.savedSchema = MysqlSavedSchema.restore(schemaConnection, this.context);
-			} else {
-				initFirstRun(connection, schemaConnection);
-			}
+			String producerClass = this.context.getProducer().getClass().getSimpleName();
+			LOGGER.info("Maxwell is booting (" + producerClass + "), starting at " + this.context.getInitialPosition());
 		} catch ( SQLException e ) {
```

Comment:
yeah, I think lots of methods in here still throw that.  `ensureMaxwellSchema` for one.